Test.cs

```diff
@@ -34,6 +56,10 @@ internal static async Task<string> doSomth()
             var logUrl = lines[i].Replace("results.json", "git-log.txt");
             var content = await querySolver.solveQuery(main + logUrl);
             var flavorData = new FlavorData(main + fileUrl, getFlavor(fileUrl), json, content);
+            foreach (var item in flavorData.results.minTimes)
+            {
+                list.Add(new GraphPointData(flavorData.commitTime.Date.ToShortDateString(), flavorData.flavor, item));
```

Comment:
`Date.ToShortDateString` is dropping the time part. we should just put `DateTimeOffset` to the `GraphPointData`, so that it gets to the JS.